recipes/code-aster/meta.yaml

```diff
@@ -0,0 +1,102 @@
+{% set name = "code-aster" %}
+{% set version = "14.6.11" %}
+{% set sha256 = "107ed98d10bc05866bdb3053f088fb0f51276ea14f7b615ec0d410a4b147ef12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - fn: {{ name }}_{{ version }}.tar.gz
+    url: https://gitlab.com/codeaster/src/-/archive/{{ version }}/src-{{ version }}.tar.gz
+    sha256: {{ sha256 }}
+  - url: https://downloads.sourceforge.net/project/prereq/metis-5.1.0_aster4.tar.gz
+    sha256: 1b295f9ce260c19d2d93fb2430be4139496c56f58f74ef7956ccd26c854fcc2e
+    folder: metis-aster
+  - url: http://graal.ens-lyon.fr/MUMPS/MUMPS_5.1.2.tar.gz
+    sha256: eb345cda145da9aea01b851d17e54e7eef08e16bfa148100ac1f7f046cd42ae9
+    folder: mumps-aster
+    patches:
+      - contrib/mumps-aster-diff.patch
```

Comment:
I see that you have correctly included the licenses for these dependencies, but can you please also write two comments explaining how the aster versions of MUMPS and METIS have been modified such that you cannot use the MUMPS and METIS packages already published on conda-forge? Thank you.